examples/http-request-error-burnrate.jsonnet

```diff
@@ -7,8 +7,7 @@ local slo = import '../slo-libsonnet/slo.libsonnet';
     // However, it is availabe on every Prometheus by default.
     metric: 'promhttp_metric_handler_requests_total',
     selectors: ['namespace="default"', 'job="fooapp"'],
-
-    errorBudget: 1 - 0.999,
+    target: 0.999,  // 99.9%
```

Comment:
for usability `99.9` would make even more sense here, no?

Reply:
Yeah, I'm pretty unsure about that. Most of the queries we run return 0-1.
@krasi-georgiev had similar comments in the past.

Reply:
I would try to keep internal implementation details decoupled from the UX. The fact that that comment is there kind of shows that it needs explaining. I don't feel too strongly about it, just seems like something that might be easier for people to grasp who are just getting started.